web/client/utils/ogc/Filter/CQL/parser.js

```diff
@@ -344,33 +397,43 @@ const buildAst = (tokens) => {
 
     return result;
 };
-module.exports = {
-    /**
-     * Parse a CQL filter. returns an object representation of the filter.
-     * For the moment this parser doesn't support WKT parsing.
-     * Example:
-     * ```
-     * const cqlFilter = "property1 = 'value1' AND property2 = 'value2'";
-     * const obj = read(cqlFilter);
-     * console.log(obj);
-     * // obj looks like this
-     * {
-     *   type: "and",
-     *   filters: [{
-     *      type: "=",
-     *      property: "property1",
-     *      value: "value1"
-     *   },{
-     *      type: "=",
-     *      property: "property1",
-     *      value: "value2"
-     *   }]
-     * }
-     * ```
-     * @memberof utils.ogc.Filter.CQL.parser
-     * @name read
-     * @param cqlFilter the cql_filter o parse
-     * @return a javascript representation of the filter.
-     */
-    read: (text) => buildAst(tokenize(text))
-};
+
+
+/**
+ * Parse a CQL filter. returns an AST object representation of the filter.
+ * For the moment this parser doesn't support WKT parsing.
```

Comment:
should this be removed or edited?
```suggestion
 * For the moment this parser doesn't support WKT parsing.
```